packages/portfolio-contract/test/planner.exo.test.ts

```diff
@@ -28,26 +32,73 @@ test('planner exo submit method', async t => {
     }),
   } as ZCF;
 
-  const mockGetPortfolio = id => null as any;
+  const board = makeFakeBoard();
+  const storage = makeFakeStorageKit('published', { sequence: true });
+  const readPublished = async path => {
+    await eventLoopIteration();
+    return marshaller.fromCapData(
+      JSON.parse(storage.getValues(`published.${path}`).at(-1) || ''),
+    );
+  };
+  const marshaller = board.getReadonlyMarshaller();
+  const makePortfolio = preparePortfolioKit(zone, {
+    usdcBrand: USDC,
+    marshaller,
+    portfoliosNode: storage.rootNode.makeChildNode('portfolios'),
+    ...({} as any),
+  });
+  const aPortfolio = makePortfolio({ portfolioId: 1 });
+  const mockGetPortfolio = _id => aPortfolio;
 
-  const shapes = makeOfferArgsShapes(USDC);
   // Create planner exo
   const makePlanner = preparePlanner(zone, {
     rebalance: mockRebalance,
     zcf: mockZcf,
     getPortfolio: mockGetPortfolio,
-    shapes,
+    shapes: makeOfferArgsShapes(USDC),
   });
 
   const planner = makePlanner();
 
   // Test submit method
+  aPortfolio.manager.setTargetAllocation({ USDN: 100n });
+
+  {
+    const { policyVersion, policyVersionAck } = await readPublished(
+      'portfolios.portfolio1',
+    );
+    t.log('targetAllocation', aPortfolio.reader.getTargetAllocation(), {
+      policyVersion,
+      policyVersionAck,
+    });
+    t.deepEqual(
+      { policyVersion, policyVersionAck },
+      { policyVersion: 1, policyVersionAck: 0 },
+    );
+  }
+
   const portfolioId = 0;
+  const amount = { brand: USDC, value: 100n };
   const plan: MovementDesc[] = [
-    { src: '<Deposit>', dest: '@agoric', amount: { brand: USDC, value: 100n } },
+    { src: '+agoric', dest: '@agoric', amount },
+    { src: '@agoric', dest: '@noble', amount },
+    { src: '@noble', dest: 'USDN', amount },
   ];
 
-  const result = await vt.when(planner.submit(portfolioId, plan));
+  t.throws(() => planner.submit(portfolioId, plan, 0), {
+    message: /expected policyVersion 1; got 0/,
+  });
```

Comment:
@gibson042 #11805 says `policyVersionAck` is "used to reject stale [transactions]" but taking that literally seems to mean we would accept >1 plan against the same policyVersion, which seems to defeat the whole purpose.